src/Robo/RoboFileAbstract.php

```diff
@@ -337,4 +337,28 @@ private function getBitbucketOauthBlock($host, $helpText = '')
         ];
     }
 
+    protected function initLogDirectory($projectName)
+    {
+        $this->say('Initialising the log directory');
+
+        $dirs = [
+            '/var/log/' . $projectName,
+            '/var/log/' . $projectName . '/application_log',
+            '/var/log/' . $projectName . '/error',
+        ];
```

Comment:
I'd rather define an abstract method or just a separate method what could be overwritten to retrieve the directories.
That way we won't need to overwrite the whole method if we have something different in a project

Reply:
done